app/views/layouts/_user_config.html.erb

```diff
@@ -95,6 +97,7 @@
         --theme-container-box-shadow: none;\
         --theme-container-border: 1px solid #e3e3e5;\
         --theme-subtle-border: 1px solid #fcfcfc;\
+        --theme-social-icon-color: #505159;\
```

Comment:
Nice job 🎊 

My only thought here is it might make sense for the minimal-light-theme to also default to transparent since it is similar to the default theme. Pinging @benhalpern if he has any thought on this.
```suggestion
```

Reply:
I actually don't think we should have any colors behind the icons on the pink theme? I feel like the bottom should match what we have at the top here? ![](https://cl.ly/8c5f413493a7/download/Image%202019-10-11%20at%203.00.58%20PM.png)

Reply:
Agree that it should be transparent in the pink theme, and also in the minimal light theme. Great job though!